src/components/Settings.js

```diff
@@ -1,39 +1,50 @@
 import React from 'react'
-import {NavLink, Route} from 'react-router-dom';
+import { NavLink, Route } from 'react-router-dom';
 import './App.css';
 import LogOut from "./user-page/logOut";
-import {avatar} from './user-page/avatar';
+import { avatar } from './user-page/avatar';
 import UnauthorizedSettings from './UnauthorizedSettings.js'
 import isAuthorized from './checker/authorizationChecker.js'
-import {getLogin} from "../helpers/getLogin";
+import { getLogin } from "../helpers/getLogin";
+import Dropdown from '../components/userMenu/Dropdown'
+import DropdownItem from '../components/userMenu/DropdownItem'
 
-function Settings(props){
-    return(
-        !isAuthorized()?
+function Settings(props) {
+    return (
+        !isAuthorized() ?
             <React.Fragment>
-                <UnauthorizedSettings/>
+                <UnauthorizedSettings />
             </React.Fragment>
-            :<React.Fragment>
-                <LogOut/>
-                <div>
-                    <NavLink to="/friends">
-                        <div className="friends-button"/>
+            : <React.Fragment>
+
+                <Dropdown label="Menu" className="dropdown">
+
+                    <NavLink to="/friends" className="dropdown-option">
+                        <DropdownItem label="Friens" />
```

Comment:
I think there should be "Friends"

Reply:
Header looks like this on my PC

![Menu](https://user-images.githubusercontent.com/32419448/70140757-1ddeb580-169e-11ea-8ba1-f9b23ad0b587.JPG)